apps/vaporgui/VaporWidgetsFwd.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+class QDoubleValidator;
+class QComboBox;
+class QLineEdit;
+
+typedef QDoubleValidator VDoubleValidator; // QDoubleValidator is buggy
```

Comment:
I don't think it's a good idea to forcefully rename one object to be another very similar object. Can we specify to use which object at each occasion?